### DISCUSSION TITLE
Show weather map?

### DISCUSSION THREAD

**Discussion by s38b35M5** [Akylas/oss-weather#118] [Q&A] [answered]
I see it mentioned in preferences and here in issues, but I never see a weather map in my app. Is there a trick to getting it to load?

Follow-up: is there a matrix community for discussion?

**Comment by 0-5-0** [comment, marked as answer]
When you click the 3 dots menu from the main OSS Weather page there should be a "map" option at the end of the list, at least this is how it shows up for me on Android.

There is no Matrix community as far as I know, the discussions section in Github was opened up 4 days ago so the community is probably a bit small to spin up and manage a Matrix instance.

**Reply by s38b35M5** [reply]
:UltraFacePalm:

I knew I was missing something obvious.

**Comment by farfromrefug** [comment]
@s38b35M5 glad you found it. no matrix community cause indeed quite a smal l app yet ;)
I have been thinking about starting something for my apps to be able to help. people more directly but also engage them in ideas and possible help (like design ideas).
not sure about the platform though. discord, matrix,... I am really open 
I use discord quite a lot for work but it is proprietary and I dont know what others are using...
